app/services/keybase_service.rb

```diff
@@ -0,0 +1,27 @@
+class KeyBaseService
```

Comment:
The class should be called `KeybaseService` if the filename is `keybase_service.rb`. Might want to double check what you want here. Is it `Key Base` or `Keybase`?